.env.example

```diff
@@ -20,4 +20,6 @@ REACT_APP_GP2_AUTH0_CLIENT_ID=sUSMSrBS3FMhb58bxiwQZOTPbZMPWIje
 REACT_APP_GP2_AUTH0_DOMAIN=dev-gp2-hub.us.auth0.com
 SQUIDEX_BASE_URL=https://cloud.squidex.io
 CONTENTFUL_SPACE_ID=5v6w5j61tndm
-CONTENTFUL_ENVIRONMENT=dev
+CONTENTFUL_ENV_ID=Development
+CONTENTFUL_MANAGEMENT_ACCESS_TOKEN=
+CONTENTFUL_MIGRATIONS_DIR=/Users/me/code/asap-hub/packages/contentful/migrations
```

Comment:
I don't think we want an absolute path to migrations passed like this? I would add the relative path to the yarn command

Reply:
Yeah I can remove this now that everything is in `contentful` package. Done.